src/python/crash_analysis/stack_parsing/stack_analyzer.py

```diff
@@ -37,6 +38,15 @@ def linkify_android_stacktrace(crash_info):
     crash_info.crash_stacktrace = temp_crash_stacktrace
 
 
+def remove_kernel_times(crash_data):
+  """Filter kernel time from crash_data."""
+  result = ''
+  strip_regex = re.compile(r'^\[\s*\d+\.\d+\]\s*')
```

Comment:
this compiled regex should be in constants.py

ANDROID_KERNEL_TIME_REGEX

Also "]\s*" should be just "]\s", i think we don't match multiple space after time as it strips more than needed.